packages/next/src/build/webpack/plugins/next-trace-entrypoints-plugin.ts

```diff
@@ -180,22 +180,24 @@ export class TraceEntryPointsPlugin implements webpack.WebpackPluginInstance {
         entryFilesMap.set(entrypoint, entryFiles)
       }
 
+      const ignores = [...TRACE_IGNORES, ...this.traceIgnores]
+
+      const ignoreFn = (path: string) => {
+        return isMatch(path, ignores, { contains: true, dot: true })
+      }
+
       // startTrace existed and callable
       if (this.turbotrace) {
         let binding = (await loadBindings()) as any
         if (
           !binding?.isWasm &&
           typeof binding.turbo.startTrace === 'function'
         ) {
-          this.chunksToTrace = [...chunksToTrace]
+          this.chunksToTrace = [...chunksToTrace].filter(ignoreFn)
```

Comment:
This doesn't look like it matches the ignore behavior as `ignore` is meant to apply to any files found during tracing.